packages/dds/matrix/src/matrix.ts

```diff
@@ -594,19 +727,76 @@ export class SharedMatrix<T = any>
 				new ObjectStoragePartition(storage, SnapshotPath.cols),
 				this.serializer,
 			);
-			const [cellData, pendingCliSeqData] = await deserializeBlob(
+			const [cellData, pendingCliSeqData, cellLastWriteTracker] = await deserializeBlob(
 				storage,
 				SnapshotPath.cells,
 				this.serializer,
 			);
 
 			this.cells = SparseArray2D.load(cellData);
 			this.pending = SparseArray2D.load(pendingCliSeqData);
+			// Old summaries will not have this info, so check that.
+			if (cellLastWriteTracker !== undefined) {
+				this.cellLastWriteTracker = SparseArray2D.load(cellLastWriteTracker);
+			}
+
+			this.setCellPolicySwitchOpSeqNumber = await deserializeBlob(
+				storage,
+				SnapshotPath.setCellConflictResolutionPolicyIsFWW,
+				this.serializer,
+			).catch((error) => {
+				// Older Snapshot or summaries taken by older clients will not have this blob.
+				if (isFileNotFoundOrAccessDeniedError(error) === true) {
```

Comment:
this feels fishy. We have examples of code in other DDSs that detects if blob is there or not, without attempting to load it and differentiate based on error. Take a look at IChannelStorageService.contains() (storage variable is of that type)

Reply:
Removed this code as now encoding in same blob as per other suggestion.